calico_containers/integrations/kubernetes/calico_kubernetes.py

```diff
@@ -0,0 +1,309 @@
+#!/bin/python
+from collections import namedtuple
+import json
+import os
+import socket
+import sys
+from subprocess import check_output, CalledProcessError, check_call
+import requests
+import sh
+
+# Append to existing env, to avoid losing PATH etc.
+# Need to edit the path here since calicoctl loads client on import.
+# TODO-PAT: This shouldn't be hardcoded
+os.environ['ETCD_AUTHORITY'] = 'kubernetes-master:6666'
+
+from calicoctl import container_add
+from pycalico.datastore import IF_PREFIX
+from pycalico.util import generate_cali_interface_name
+
+CALICOCTL_PATH = os.environ.get('CALICOCTL_PATH', '/home/vagrant/calicoctl')
```

Comment:
Yep, that's v1.1. (More specifically, we're going to pull the common logic out of calicoctl.py and into a common module, leaving calicoctl as just a CLI wrapper)